scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/simulation/SimulationFacade.java

```diff
@@ -129,7 +134,7 @@ public void removeActivitiesFromSimulation(final Collection<SchedulingActivityDi
       final var oldInsertedActivities = new HashMap<>(insertedActivities);
       insertedActivities.clear();
       planActDirectiveIdToSimulationActivityDirectiveId.clear();
-      driver = new ResumableSimulationDriver<>(missionModel, planningHorizon.getAerieHorizonDuration());
+      driver.initSimulation();
```

Comment:
I'm not sure whether or not this underlies the test failures - but if we want to be extra sure that we're not altering scheduling semantics with this hotfix PR, we can make the following more conservative change:

```suggestion
      driver.close();
      driver = new ResumableSimulationDriver<>(missionModel, planningHorizon.getAerieHorizonDuration());
```